### PR TITLE
increase proportion of capitalised words required for matching

### PR DESCRIPTION
**Pull request by AdamDM**
Simple number change and some refactoring for line length

### CODE REVIEW

**Comment by dragospitica** [comment]
& code style... pay more attention to the commit message :D

File: src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java [hunk 1]
```diff
@@ -53,10 +54,12 @@ public static DescriptionMatchingScorer makeScorer(){
...
-        Score score = Score.nullScore();
-
-        score = score(subject, candidate);
-        desc.appendText("%s (%s) scored: %s", candidate.getTitle(), candidate.getCanonicalUri(), score);
+        Score score = score(subject, candidate);
+        desc.appendText(
+                "%s (%s) scored: %s",
+                candidate.getTitle(),
+                candidate.getCanonicalUri(),
+                score);
```
Comment (jamiesprax) [inline src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java]: the ); should be on the next line

File: src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java [hunk 2]
```diff
@@ -69,9 +72,20 @@ private boolean descriptionMatch(Item subject, Item candidate) {
         Set<String> candidateList = descriptionToProcessedList(candidate.getDescription());
         Set<String> subjectList = descriptionToProcessedList(subject.getDescription());
 
-        candidateList.retainAll(subjectList);
+        Set<String> allCapitalisedWordsList = Sets.newHashSet(candidateList);
+
+        allCapitalisedWordsList.addAll(
+                subjectList.stream()
+                .filter(word -> !candidateList.contains(word))
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java]: Indentation

File: src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java [hunk 3]
```diff
@@ -69,9 +72,20 @@ private boolean descriptionMatch(Item subject, Item candidate) {
         Set<String> candidateList = descriptionToProcessedList(candidate.getDescription());
         Set<String> subjectList = descriptionToProcessedList(subject.getDescription());
 
-        candidateList.retainAll(subjectList);
+        Set<String> allCapitalisedWordsList = Sets.newHashSet(candidateList);
+
+        allCapitalisedWordsList.addAll(
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/equiv/scorers/DescriptionMatchingScorer.java]: This logic needs to be simplified. Let's speak f2f.

**Pull request merged**